### PR TITLE
Console batch tool

### PR DESCRIPTION
**Pull request by GentleHedgehog**
Hello, I've created a new branch (on the release branch base) to use the batch tool (Do i need to merge it to a "new-feature" branch instead of the release branch? ).

An example of using an application with two arguments:
AwesomeBumpConsole.exe /path/to/image /path/to/result_folder 

I tested this version of the application with the following environment:
Windows 7 x64, 
Qt 5.7.0 (qt-opensource-windows-x86-mingw530-5.7.0)
Graphics Chipset - AMD Radeon R7 200 Series (driver version 17.7.1)

### CODE REVIEW

**Comment by kmkolasinski** [comment]
Hi, thank you very much for your contribution, 👍   I will take a look on it, but please give few days. I will check the code and test it.

File: Sources/main.cpp [hunk 1]
```diff
@@ -212,23 +223,28 @@ int main(int argc, char *argv[])
     QApplication app(argc, argv);
 
 
-
     regABSliderDelegates();
     regABColorDelegates();
 
-//    qInstallMessageHandler(customMessageHandler);
+#ifdef CONVERT_TO_CONSOLE
```
Comment (kmkolasinski) [inline Sources/main.cpp:229]: What is the purpose of this code?

**Review by kmkolasinski — COMMENTED** [review]
I cannot make it work on Linux: Running `./AwesomeBump ~/Imagens/test.png ~/Imagens/test` opens GUI as normal. Could you please explain in more details the usage of this tool ?

**Comment by kmkolasinski** [comment]
If this will work I think it would be nice to add possibility to choose project settings as additional input: `./AwesomeBump ~/Imagens/test.png ~/Imagens/test Configs/49_wood.ini`

**Comment by GentleHedgehog** [comment]
Sorry, this is redundant code, and I forgot to remove it. I did not try to run this application in Linux. I will try to port it to Linux and soon upload it to my branch

**Comment by kmkolasinski** [comment]
The problem is that I'm able to compile and run AB on linux but there is no effect when arguments are passed. When will you check this ?

**Comment by GentleHedgehog** [comment]
I'll check it in a few days. But I do not know if it will work on my laptop configuration (thinkpad e531).

**Comment by kmkolasinski** [comment]
Ok, we are not in hurry. It should work in case of any problem please give me a feedback. What linux do you use? You will have to install AB with GL330 mode.

**Comment by GentleHedgehog** [comment]
This is a source about my processor and integrated graphics - http://www.cpu-world.com/CPUs/Pentium_Dual-Core/Intel-Pentium%20Mobile%202030M.html . My OS - Ubuntu 12.04, and till this moment I couldn't update MESA drivers to OpenGl 3.3 or higher.

**Comment by kmkolasinski** [comment]
Ok don't waste your time with ubuntu then. I will try to check it on my windows 10, but I will need couple of days.

**Pull request closed without merging**